spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/annotation/LoadBalancerClientConfiguration.java

```diff
@@ -109,23 +86,6 @@ public ServiceInstanceListSupplier discoveryClientServiceInstanceListSupplier(
 					.withCaching().build(context);
 		}
 
-		@Bean
```

Comment:
I think it's just to replace the deprecated class, not to delete it completely

Reply:
no